cmd/current.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tfversion/tfversion/pkg/current"
+	"github.com/tfversion/tfversion/pkg/helpers"
+)
+
+const (
+	currentExample = "# Print the current active version of Terraform\n" +
+		"tfversion current"
+)
+
+var (
+	currentCmd = &cobra.Command{
+		Use:     "current",
+		Short:   "Print the current active version of Terraform",
+		Example: currentExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := current.CheckCurrentVersion()
+			if err != nil {
+				helpers.ExitWithError("could not determine active Terraform version", err)
+				os.Exit(1)
```

Comment:
When using `helpers.ExitWithError`, no `os.Exit(1)` is needed anymore.

Reply:
Resolved in fd905807e0e559b2d4bb6743350e353e17ebc392